src/service/api.tsx

```diff
@@ -117,26 +117,30 @@ const api = {
     return obj.data;
   },
 
-  async getPopularQueryList() {
-    const obj = await axios.get(`movie/popular?${API_KEY}&query`);
+  async getPopularQueryList(page: number = 1) {
+    const obj = await axios.get(`movie/popular?${API_KEY}&page=${page}`);
     return obj.data.results;
   },
 
-  async getTopRatedList() {
-    const obj = await axios.get(
-      `movie/top_rated?translations&${API_KEY}&region=US`
-    );
+  async getTopRatedList(page: number = 1) {
+    const obj = await axios.get(`movie/top_rated?translations&${API_KEY}&region=US&page=${page}`);
     return obj.data.results;
   },
 
-  async getSearchList(query: string) {
-    let obj = await axios.get(`search/movie?${API_KEY}&query=${query}`);
+  async getSearchList(query: string, page: number = 1) {
+    let obj = await axios.get(`search/movie?${API_KEY}&query=${query}&page=${page}`);
     return obj.data.results;
   },
 
-  changeImgLinks(url: string, size: string) {
+  changeImgLinks(url: string, size: string = 'w500') {
     return `${API_IMG_URL}${size}${url}`;
-  }
-};
+  },
+
+  changeListByPagination(arr: {}[], page: number = 1): {}[] {
+    return arr.length < 6
+      ? arr
+      : arr.slice(6 * (page - 1), 6 * page);
+  },
```

Comment:
Already left comment for this code
add types 
Array<T>
{}[] - is NOT a valid type